### PR TITLE
Allow setting an api key

### PR DESCRIPTION
**Pull request by Hayden289**
Add the ability to set an api key for google maps

### CODE REVIEW

File: geolib.js [hunk 1]
```diff
@@ -703,9 +703,10 @@
 		 *
 		 *  @return     Array [{lat:#lat, lng:#lng, elev:#elev},....]}
 		 */
-		setBusinessSpecificParameters: function(clientId, privateKey){
+		setBusinessSpecificParameters: function(clientId, privateKey, key){
 			gm.config('google-client-id', clientId);
-			gm.config('google-private-key', privateKey);
+      gm.config('google-private-key', privateKey);
```
Comment (jwenzler) [inline geolib.js]: Fix that spacing! Haha, jk

How are you Hayden :)

Comment (Hayden289) [reply geolib.js]: That's weird it looks fine in my IDE even after I pulled, I had to fix it in the web editor here 😖 
I'm doing good! How are you?

Comment (jwenzler) [reply geolib.js]: Haha, I've started using VSCode and love it (used to use Atom and had some strange issues sometimes.

I'm good, I saw this PR in my email and was surprised then noticed it was you.

Comment (Hayden289) [reply geolib.js]: Yeah I did a double take when your review popped up haha

Comment (ThomasWilliams) [reply geolib.js]: 👋 Hi John!

File: geolib.js [hunk 2]
```diff
@@ -703,9 +703,10 @@
 		 *
 		 *  @return     Array [{lat:#lat, lng:#lng, elev:#elev},....]}
 		 */
-		setBusinessSpecificParameters: function(clientId, privateKey){
+		setBusinessSpecificParameters: function(clientId, privateKey, key){
```
Comment (ThomasWilliams) [inline geolib.js]: I left a similar review on the PR in the other repo already, but naming this variable "key" doesn't tell me how this is different from `privateKey`.

File: geolib.js (hunk 1 above)
Comment (jwenzler) [reply geolib.js]: Hey Thomas! Hope things are going great for you.

**Pull request merged**